jcefmaven/src/main/java/me/friwi/jcefmaven/impl/step/fetch/PackageDownloader.java

```diff
@@ -27,7 +27,7 @@ public class PackageDownloader {
     private static final int BUFFER_SIZE = 16 * 1024;
 
     public static void downloadNatives(CefBuildInfo info, EnumPlatform platform, File destination,
-                                       Consumer<Float> progressConsumer, Set<String> mirrors) throws IOException {
+                                       Consumer<Float> progressConsumer, Collection<String> mirrors) throws IOException {
```

Comment:
Technically, this forms part of the public API, so rather than changing this to a `List<String>` (which would break anyone's existing use of this with a `Set<>`), I've relaxed the restriction to be just a `Collection<String>`

An alternative solution I considered was to have separate overloads for `List<>` and `Set<>`, and mark the `Set<>` version as deprecated to encourage people to move to the more predictable version - I'd be happy to make this change if you think it is more appropriate.

Reply:
Both options would be valid, but I think you chose the best one for compatibility. :)